sentinel-cluster/sentinel-cluster-server-default/src/main/java/com/alibaba/csp/sentinel/cluster/server/connection/ScanIdleConnectionTask.java

```diff
@@ -32,9 +32,8 @@ public void run() {
             for (Connection conn : connections) {
                 if ((now - conn.getLastReadTime()) > idleTimeMillis) {
                     RecordLog.info(
-                        String.format("[ScanIdleConnectionTask] The connection <%s:%d> has been idle for <%d>s. "
-                            + "It will be closed now.", conn.getRemoteIP(), conn.getRemotePort(), idleSeconds)
-                    );
+                        "[ScanIdleConnectionTask] The connection <{}:{}> has been idle for <{}>s. "
+                            + "It will be closed now.", conn.getRemoteIP(), conn.getRemotePort(), idleSeconds);
```

Comment:
A unnecessary concatenating is still kept here.

Reply:
is this ok?
```java
RecordLog.info("[ScanIdleConnectionTask] The connection <{}:{}> has been idle for <{}>s. It will be closed now.", 
    conn.getRemoteIP(), conn.getRemotePort(), idleSeconds);
```

Reply:
Yes, exactly